src/stylesheets/components/_article.scss

```diff
@@ -1,14 +1,38 @@
+/*--- Article Styles ---*/
+
 article {
   position: relative;
-  margin: $base-spacing-unit-extra-large 0 $base-spacing-unit-small;
-  @include media($large) {
-    flex: 1;
-    margin-top: ($base-spacing-unit-large * 2);
-  }
 
   @media print {
     p, blockquote, li {
       page-break-inside: avoid;
     }
   }
 }
+
+article * .container {
```

Comment:
Are there any particular reasons why you are not nesting these styles?

Reply:
Readability is not great when nested